00_ES6/04 redux_saga/src/actions/index.js

```diff
@@ -22,3 +23,31 @@ export const canEnrollRequest = (nickname, room) => (dispatch) => {
   });
   // For the sake of the sample no Error handling, we could add it here
 }
+
+export const EnrollRoomRequest = (nickname, room) => ({
+  type: actionIds.ENROLL_ROOM_REQUEST,
+  payload: { nickname, room }
+});
+
+export const disconnectRoomRequest = () => ({
+  type: actionIds.DISCONNECT,  
+});
+
+export const onDisconnect = () => ({
```

Comment:
is this  action used?

Reply:
Is there, but we are not going to implement the hooks, can be lived as an excercise